loop-de-loop.ipynb

```diff
@@ -54,7 +54,9 @@
    "cell_type": "markdown",
```

Comment:
<p><strong>(b)</strong> and <strong>(c)</strong> also work. <strong>(d)</strong>&nbsp;was supposed to, but has an unintended error (<code>sumlist</code>&nbsp;is not initialized).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UMD-ENMA165/loop-de-loop/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>